Applications/CommandLineInterface/src/selxSuperElastix.cxx

```diff
@@ -159,15 +173,12 @@ main( int ac, char * av[] )
 
         // since we do not know which reader type we should instantiate for input "name",
         // we ask SuperElastix for a reader that matches the type of the source component "name"
-        logger->Log( selx::LogLevel::INF, "Preparing input " + name + " ..." );
+        logger->Log( selx::LogLevel::INF, "Preparing input '" + name + "': " + path + " ..." );
         selx::AnyFileReader::Pointer reader = superElastixFilter->GetInputFileReader( name );
         reader->SetFileName( path );
         superElastixFilter->SetInput( name, reader->GetOutput() );
         fileReaders.push_back( reader );
-
-        logger->Log( selx::LogLevel::INF, "Preparing input " + name +  "... Done" );
-        std::cout << "Input data " << index << " " << name << " : " << path << "\n";
-        ++index;
+        logger->Log( selx::LogLevel::INF, "Preparing input '" + name + "': " + path + " ... Done" );
```

Comment:
What does "preparing" mean? Loading? Validating? Both? Can we find a more descriptive word?

Reply:
Yes, I like to be more descriptive, but in this case it is neither loading nor validating the files. It is just passing the filename without any checks. The actual loading happens when/if the itk pipeline triggers an update on the reader.